src/components/auth/ProfileModal/contents/notifications/TelegramNotificationContent.tsx

```diff
@@ -115,13 +116,15 @@ function ConnectTelegramButton({ address }: ContentProps) {
       address,
     })
   const [openedTelegramBotLink, setOpenedTelegramBotLink] = useState(false)
+  const sendEvent = useAnalytics((state) => state.sendEvent)
```

Comment:
there are 2 types of calling that are used,
`useSendEvent` and `useAnalytics(state => state.sendEvent)`
well, both are same, but I think its better to use useSendEvent everywhere